testing/checkin/runner/ops_io.rs

```diff
@@ -1,3 +1,4 @@
+use deno_core::anyhow::Error;
```

Comment:
This bites me all the time -- run `tools/lint.ts --fix` :)

Reply:
@dsherret suggested to put an empty line after copyright and that will make rust-analyzer behave